### PR TITLE
Create Dockerfile

### PR DESCRIPTION
**Pull request by michiomochi**
docker image URL: https://hub.docker.com/r/bitjourney/plantuml-service/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM openjdk:8
```
Comment (michiomochi) [inline Dockerfile]: see: https://hub.docker.com/_/openjdk/

**Comment by michiomochi** [comment]
@bitjourney/engineers Please review 🙏

File: bin/release_docker_image.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -xe
+
+BASE_TAG=bitjourney/plantuml-service
+TAG=${BASE_TAG}:1.3.3
+
+docker --version
+docker login
```
Comment (gfx) [inline bin/release_docker_image.sh]: Don't login in the release engineering script. It should already be logged-in.

Comment (gfx) [reply bin/release_docker_image.sh]: In addition, can you merge the script to `./gradlew release` ask?

File: Dockerfile (hunk 1 above)
Comment (gfx) [reply Dockerfile]: Can you comment the URL to Dockerfile?

File: bin/release_docker_image.sh (hunk 2 above)
Comment (michiomochi) [reply bin/release_docker_image.sh]: >  Don't login in the release engineering script. It should already be logged-in.

Why should not login? 
I think it is difficult to force already logged in.
This release script will fail. If not already logged in. So, I force login by `docker login` command.
Is there any other good way?

Comment (michiomochi) [reply bin/release_docker_image.sh]: > In addition, can you merge the script to ./gradlew release ask?

OK, I'll try it!

File: Dockerfile (hunk 1 above)
Comment (michiomochi) [reply Dockerfile]: Commented!
31825db

File: bin/release_docker_image.sh (hunk 2 above)
Comment (gfx) [reply bin/release_docker_image.sh]: As of docker v18.06.0, `docker login` does no-op if it is already logged-in. So the line is not bothersome.

**Pull request merged**